openjdk/mmtkBarrierSetAssembler_x86.cpp

```diff
@@ -76,34 +76,13 @@ void MMTkBarrierSetAssembler::eden_allocate(MacroAssembler* masm, Register threa
       fatal("unimplemented allocator fastpath\n");
     }
 
-    // Calculat offsets of top and end. We now assume we are using bump pointer.
-    int allocator_base_offset;
+    // Calculate offsets of top and end. We now assume we are using bump pointer.
```

Comment:
Can you remove the part 'We now assume we are using bump pointer'? That is clearly outdated.

Reply:
Yep will do :+1:

Reply:
Should I remove this extra check as well?

https://github.com/mmtk/mmtk-openjdk/pull/158/files#diff-ca716ee22f1d39841309c8bfd97ab45c30ad2df222466e0b1b153ea6dbcb23e9R74-R77

Reply:
I cannot see what that link refers to.

Reply:
Sorry I have updated it

Reply:
Yeah. Feels like it should be removed.

Reply:
Done